include/rcpputils/macros.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCPPUTILS__MACROS_HPP_
+#define RCPPUTILS__MACROS_HPP_
+
+#define RCPPUTILS_STRING_JOIN(arg1, arg2) RCPPUTILS_DO_STRING_JOIN(arg1, arg2)
+#define RCPPUTILS_DO_STRING_JOIN(arg1, arg2) arg1 ## arg2
```

Comment:
this could live in `rcutils` `macros.h`, but it doesn't matter much

Reply:
That's annoying. But I agree.